crates/uv-types/src/build_options.rs

```diff
@@ -211,6 +211,37 @@ impl NoBuild {
     }
 }
 
+#[derive(Debug, Default, Clone, Hash, Eq, PartialEq)]
+pub enum IndexStrategy {
+    /// Only use results from the first index that returns a match for a given package.
+    ///
+    /// While this differs from pip's behavior, it's the default index strategy as it's the most
+    /// secure.
+    #[default]
+    FirstMatch,
```

Comment:
Is a match just "package name" or "package name and version range"?

Reply:
Package name.